src/Material.cc

```diff
@@ -205,61 +202,49 @@ Errors Material::Load(sdf::ElementPtr _sdf)
 }
 
 //////////////////////////////////////////////////
-ignition::math::Color Material::Ambient() const
+gz::math::Color Material::Ambient() const
 {
   return this->dataPtr->ambient;
 }
 
 //////////////////////////////////////////////////
-void Material::SetAmbient(const ignition::math::Color &_color)
+void Material::SetAmbient(const gz::math::Color &_color)
 {
   this->dataPtr->ambient = _color;
 }
 
 //////////////////////////////////////////////////
-ignition::math::Color Material::Diffuse() const
+gz::math::Color Material::Diffuse() const
 {
   return this->dataPtr->diffuse;
 }
 
 //////////////////////////////////////////////////
-void Material::SetDiffuse(const ignition::math::Color &_color)
+void Material::SetDiffuse(const gz::math::Color &_color)
 {
   this->dataPtr->diffuse = _color;
 }
 
 //////////////////////////////////////////////////
-ignition::math::Color Material::Specular() const
+gz::math::Color Material::Specular() const
 {
   return this->dataPtr->specular;
 }
 
 //////////////////////////////////////////////////
-void Material::SetSpecular(const ignition::math::Color &_color)
+void Material::SetSpecular(const gz::math::Color &_color)
 {
   this->dataPtr->specular = _color;
 }
 
 //////////////////////////////////////////////////
-double Material::Shininess() const
-{
-  return this->dataPtr->shininess;
-}
-
-//////////////////////////////////////////////////
-void Material::SetShininess(const double _shininess)
-{
-  this->dataPtr->shininess = _shininess;
-}
```

Comment:
Oops, we shouldn't remove this

Reply:
:O, how did that get removed, I think it's because SDFormat is due for a rebase?

Reply:
Rebased! Change doesn't appear anymore.

Also I took the liberty of cleaning up some dangling github links from the rebase
https://github.com/gazebosim/sdformat/pull/1022/commits/8920295f0f15f7fc87411589d7208dfa205bfc6d